container/rules/docker_toolchains.bzl

```diff
@@ -62,16 +63,43 @@ def _language_tool_layer_impl(ctx, symlinks=None, env=None, tars=None,
   files = files or ctx.files.files
   packages = packages or ctx.attr.packages
   additional_repos = additional_repos or ctx.attr.additional_repos
-  installables_tars = installables_tars or []
-  if installables_tars == [] and ctx.file.installables_tar:
-    installables_tars = [ctx.file.installables_tar]
   installation_cleanup_commands = installation_cleanup_commands or ctx.attr.installation_cleanup_commands
+  installables_tar = None
+
+  # If a link to the tarball of deb packages is passed in through
+  # --action_env=DOWNLOADABLE_TAR=<link>, use that one.
+  if "DOWNLOADABLE_TAR" in ctx.configuration.default_shell_env:
+    if "DOWNLOADABLE_TAR_SHA256" not in ctx.configuration.default_shell_env:
+      fail("'DOWNLOADABLE_TAR_SHA256' must be specified when 'DOWNLOADABLE_TAR' is specified.")
+    installables_tar = ctx.actions.declare_file(ctx.attr.name + "-packages.tar")
+
+    wget_command = """\
+#!/bin/bash
+set -e
+# Execute 'wget' command inside a docker container in case the host does not have 'wget' installed.
+cid=$(docker run -w="/" -e DOWNLOADABLE_TAR -d --privileged launcher.gcr.io/google/rbe-debian8:latest sh -c "wget $DOWNLOADABLE_TAR --no-verbose -O /tmp/debs.tar")
```

Comment:
I'm a little unsure about reusing rbe-debian8 container here. The reason why I have to do wget in a container is because the bazel container on GCB doesn't have wget installed. So I just picked a container that have wget in it, and the first one come to my mind is our container. Let me know if you have any ideas.

Reply:
don't like this, we instead need to have a container on GCB that does have wget installed and make wget part of the required tools for our container rules. That also means we will need a custom container for GCB that has bazel + wget. I'm pretty sure we have done this with Yu in the past

Reply:
Yes, to use a custom docker container with GCB you can reference [Creating Custom Build Steps](https://cloud.google.com/container-builder/docs/create-custom-build-steps). I can help you with that.